app/routines/admin/search_security_log.rb

```diff
@@ -0,0 +1,153 @@
+# Routine for searching the Security Log by admins
+#
+# Caller provides a query and some options.  The query follows the rules of
+# https://github.com/bruce/keyword_search , e.g.:
+#
+#   "user:jps,richb" --> returns the security log for the "jps" and "richb" users
+#   "application:Tutor" --> returns the security log for the Tutor application
```

Comment:
Could lean on Chronic and just go with it:

Query: `time:'1 day ago to 2 days ago'`

```ruby
[2] pry(main)> '1 day ago to 2 days ago'.split('to').map{|str| Chronic.parse(str)}.sort
=> [2016-05-08 17:58:35 -0700, 2016-05-09 17:58:35 -0700]
```